docs/UserGuide.md

```diff
@@ -13,7 +13,131 @@
 
 ## Features 
 
-{Give detailed description of each feature}
+
+
+
+
+
+
+
+## 3. Academic
+Have you ever had to dig through tons of emails to look for the contact of a particular TA 
+or sweat over your calculator when trying to estimate your CAPS this semester? 
+Academic tracker provides a convenient experience where you can store 
+all the information you need in one place! To access the academic tracker, 
+follow the instructions below.
+
+###3.1 Entering Academic Mode : Entering Academic Mode : 
```

Comment:
There should be a space after # so that the heading is displayed properly on GitHub
i.e. `### 3.1`